maps/wasteland/market.lua

```diff
@@ -80,7 +84,7 @@ local upgrade_functions = {
     end,
     -- Set Spawn Point
     [7] = function(town_center, player)
-        local this = ScenarioTable.get_table()
+        local this = global.tokens.maps_wasteland_table
```

Comment:
Is this always safe + worth the performance gain to break the pattern?

Reply:
Better performance in theory: [https://gist.github.com/ZwerOxotnik/9567b4975b4e32db0ca70a0ea988dcff](https://gist.github.com/ZwerOxotnik/9567b4975b4e32db0ca70a0ea988dcff)  
I can't guarantee safety since I'm not fully aware how the stuff is used in the `ScenarioTable`, but this project uses only 1 "map" and this file meant for the map only as far I know.

Reply:
Okay